src/process.ts

```diff
@@ -47,7 +52,7 @@ export class ChildProcessWrapper {
     if (isWin32) {
       return this.execSync(`taskkill /F /PID ${pid}`);
     } else {
-      return this.execSync(`kill -SIGINT ${pid}`);
+      return this.execSync(`kill -INT ${pid}`);
```

Comment:
The child_process package uses the /bin/sh shell by default, where trying to use SIGINT fails. INT is the equivalent and works on both macos and linux.

Reply:
Thanks, lgtm !